packages/sdks-tests/src/e2e-tests/custom-breakpoints.spec.ts

```diff
@@ -115,34 +115,22 @@ breakpoints: {
     });
 
     test('extra small mobile size', async ({ page, sdk }) => {
-      test.skip(excludeGen2(sdk));
+      test.skip(
+        excludeTestFor({ reactNative: true }, sdk),
+        'Custom breakpoints not implemented in React Native'
+      );
       await page.setViewportSize({ width: 320, height: 1000 });
       await page.goto('/custom-breakpoints');
 
+      const expectedTextColor = 'rgb(25, 201, 216)'; // blueish text color
+
       const breakpointsPara = page.locator(`text=BREAKPOINTS 500 - 800`);
+      await expect(breakpointsPara).toHaveCSS('color', expectedTextColor);
 
-      await expect(breakpointsPara).toHaveCSS('color', 'rgb(25, 201, 216)');
+      const expectedColumnTextColor = 'rgb(25, 201, 216)'; // blueish text color
 
       const column2 = page.locator(`text=Column 2`);
-
-      await expect(column2).toHaveCSS('color', 'rgb(25, 201, 216)');
-
-      // Skipping this image test for react-native.
-      // Its difficult to locate the image in react-native as css selectors don't work as expected.
-      if (!checkIsRN) {
-        const image = page.locator(`.builder-block:has(img.builder-image)`);
-
-        const expectedImageCss: Record<string, string> = {
-          display: 'flex',
-          width: '156px',
-          'max-width': '250px',
-        };
-
-        await expectStylesForElement({
-          locator: image,
-          expected: expectedImageCss,
-        });
-      }
```

Comment:
why did you remove this test?

Reply:
It was not being executed before due to the condition `if(!checkRn)` where checkRn was a function. I have corrected the test case now.